### ISSUE TITLE
[AST] Implement inet data type in ast

### ISSUE THREAD

**Issue by seonghun-dev** [gluesql/gluesql#814]
# Description
Implements ast for `inet` data type

> inet
> The inet type holds an IPv4 or IPv6 host address, and optionally its subnet, all in one field. The subnet is represented by the number of network address bits present in the host address (the “netmask”). If the netmask is 32 and the address is IPv4, then the value does not indicate a subnet, only a single host. In IPv6, the address length is 128 bits, so 128 bits specify a unique host address. Note that if you want to accept only networks, you should use the cidr type rather than inet.
>
>
> The input format for this type is address/y where address is an IPv4 or IPv6 address and y is the number of bits in the netmask. If the /y portion is omitted, the netmask is taken to be 32 for IPv4 or 128 for IPv6, so the value represents just a single host. On display, the /y portion is suppressed if the netmask specifies a single host.
> IPv4 and IPv6 hosts and networks

## Task
In AST module add `inet`  data_type

The following tests should be passed:

``` rust
test_case!(inet, async move {

    let test_cases = [
        ("CREATE TABLE INET (inet_field INET)", Ok(Payload::Create)),
        (
            r#"INSERT INTO UUID VALUES
            ('127.0.0.1'),
            ('118.235.6.252'),
            Ok(Payload::Insert(2)),
        ),
        (
             r#"SELECT inet_field AS inet_field FROM INET;"#,
            Ok(select!(
                uuid_field
                Uuid;
                parse_inet("192.168.100.128/25");
                Ipv4Addr::new(118, 223, 6, 252)
            )),
        )
    ];

    for (sql, expected) in test_cases {
        test!(expected, sql);
    }
});

```
## Reference
https://www.postgresql.org/docs/current/datatype-net-types.html
https://www.postgresql.org/docs/9.3/functions-net.html
https://cloud.google.com/bigquery/docs/reference/standard-sql/net_functions
https://doc.rust-lang.org/std/net/index.html

**Comment by pythonbrad** [comment]
Hi, i am interested by this issue. But before, i want know if all the input of this table should be take in account ?
https://www.postgresql.org/docs/current/datatype-net-types.html#DATATYPE-NET-CIDR-TABLE

**Comment by panarch** [comment]
> Hi, i am interested by this issue. But before, i want know if all the input of this table should be take in account ? https://www.postgresql.org/docs/current/datatype-net-types.html#DATATYPE-NET-CIDR-TABLE

That's up to you. we can start with minimal support at first.
Then, we can expand it later.

**Comment by seonghun-dev** [comment]
I don't think I'm going to contribute to this issue any more. Thank you for taking on this issue.

**Comment by panarch** [comment]
resolved in #1080
